checkov/arm/checks/resource/StorageAccountsUseReplication.py

```diff
@@ -0,0 +1,24 @@
+from checkov.common.models.enums import CheckCategories
+from checkov.arm.base_resource_value_check import BaseResourceValueCheck
+from typing import Any, List
+
+
+class StorageAccountsUseReplication(BaseResourceValueCheck):
+    def __init__(self) -> None:
+        name = "Ensure that Storage Accounts use replication"
+        id = "CKV_AZURE_206"
+        supported_resources = ("Microsoft.Storage/storageAccounts",)
+        categories = (CheckCategories.BACKUP_AND_RECOVERY,)
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources,)
+
+    def get_inspected_key(self) -> str:
+        return "sku/name"
```

Comment:
In the check name, we are looking for `Storage Accounts use replication`
But we are checking the `sku/name`, and I don't think it is correct.

Reply:
in the property sku/name every possible value refers to a specific replication type

Reply:
@tsmithv11 Can we rely on the SKU name?
Or should this check be a graph check as well?

Reply:
According to the docs, SKU is good enough: https://learn.microsoft.com/en-us/azure/storage/common/redundancy-migration?tabs=azure-cli